eth/tracers/api.go

```diff
@@ -911,9 +1126,22 @@ func (api *API) traceTx(ctx context.Context, message core.Message, txctx *Contex
 	// Call Prepare to clear out the statedb access list
 	statedb.Prepare(txctx.TxHash, txctx.TxIndex)
 
-	result, err := core.ApplyMessage(vmenv, message, new(core.GasPool).AddGas(message.Gas()))
-	if err != nil {
-		return nil, fmt.Errorf("tracing failed: %w", err)
+	var result *core.ExecutionResult
+
+	if config.BorTx == nil {
+		config.BorTx = newBoolPtr(false)
```

Comment:
Maybe I am missing something, why do we need `newBoolPtr`?

Reply:
Hey, config.BorTx is a *bool and we cannot define it directly AFAIK. So instead of using 3 same lines at multple places. I created a func to give *bool from bool.

Reply:
Yes, I too actually was going with `bool`, but then also wanted to keep the same standard as before, therefore used `*bool`.